myhpi/search/templates/search/search_result.html

```diff
@@ -9,7 +9,7 @@ <h5 class="text-break">{{ result.title }}</h5>
                 {{ result.specific.date|date:"SHORT_DATE_FORMAT" }}
             </time>
         </hgroup>
-        <span class="text-dark">
+        <span class="search-preview">
             {% if result.search_description %}
```

Comment:
The border at the bottom of the menu seems to be gone, which sometimes makes it hard to differentiate it from the background:

![grafik](https://github.com/user-attachments/assets/46dec71d-6d20-4320-aaee-9234b19eeb0b)

This is the old design:

![grafik](https://github.com/user-attachments/assets/a09c9d0c-90c4-466b-9443-2ad41d41c506)